rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -1750,6 +1752,49 @@ static NEVER_INLINE error_code savedata_op(ppu_thread& ppu, u32 operation, u32 v
 
 	fileGet->excSize = 0;
 
+	// show indicator for automatic save or auto load interactions if the game requests it (statSet->indicator)
+	const bool show_auto_indicator = operation <= SAVEDATA_OP_LIST_AUTO_LOAD && statSet && statSet->indicator && g_cfg.misc.show_autosave_autoload_hint;
+
+	if (show_auto_indicator)
+	{
+		auto msg_text = localized_string_id::INVALID;
+
+		if (operation == SAVEDATA_OP_AUTO_SAVE || operation == SAVEDATA_OP_LIST_AUTO_SAVE)
+		{
+			msg_text = localized_string_id::CELL_SAVEDATA_AUTOSAVE;
+		}
+		else if (operation == SAVEDATA_OP_AUTO_LOAD || operation == SAVEDATA_OP_LIST_AUTO_LOAD)
+		{
+			msg_text = localized_string_id::CELL_SAVEDATA_AUTOLOAD;
+		}
+
+		auto msg_location = rsx::overlays::message_pin_location::top_left;
+
+		switch (statSet->indicator->dispPosition & 0x0F)
+		{
+		case CELL_SAVEDATA_INDICATORPOS_LOWER_RIGHT:
+			msg_location = rsx::overlays::message_pin_location::bottom_right;
+			break;
+		case CELL_SAVEDATA_INDICATORPOS_LOWER_LEFT:
+			msg_location = rsx::overlays::message_pin_location::bottom_left;
+			break;
+		case CELL_SAVEDATA_INDICATORPOS_UPPER_RIGHT:
+			msg_location = rsx::overlays::message_pin_location::top_right;
+			break;
+		case CELL_SAVEDATA_INDICATORPOS_UPPER_LEFT:
+		default:
```

Comment:
Maybe also add CELL_SAVEDATA_INDICATORPOS_CENTER here to make it more explicit that this wasn't forgotten.